supertokens_python/recipe/thirdpartypasswordless/recipe.py

```diff
@@ -172,17 +207,20 @@ def apis_override_third_party(_: ThirdPartyAPIInterface) -> ThirdPartyAPIInterfa
             self.third_party_recipe: Union[ThirdPartyRecipe, None] = third_party_recipe
         else:
             self.third_party_recipe: Union[ThirdPartyRecipe, None] = None
+
             if len(self.config.providers) != 0:
+                tp_ingredients = ThirdPartyIngredients(None)
```

Comment:
Need to pass this email delivery as well.

Reply:
Done.